### PR TITLE
SN-API-51 add socket

### PR DESCRIPTION
**Pull request by Aretemal**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -15,3 +15,7 @@ jobs:
           echo "${{ secrets.DOCKERPW }}" | docker login -u "aretemal" --password-stdin
           docker image build -t aretemal/sn-api:latest .
           docker push aretemal/sn-api:latest
+      - name: Build MySQL container
+        run: |
+          echo "${{ secrets.DOCKERPW }}" | docker login -u "aretemal" --password-stdin
+          docker run --name sn-db -p 3000:3306 -e MYSQL_ROOT_PASSWORD=root MYSQL_DATABASE=database_development_socialnetwork -d mysql:tag
```
Comment (Konstantsin) [inline .github/workflows/main.yml:21]: Could you let me know why you're trying to run a container? 
Do github actions allow you to do that? 
How will you use it?

File: src/controllers/FollowController.js [hunk 2]
```diff
@@ -42,20 +42,20 @@ class FollowController {
...
+    const friends = await FollowService.getFriends(req.user.id, req.params.count, next);
+    req.serializer = new CollectionSerializer(friends, { serializerType: UserSerializer });
     next();
   }
 
   async getSubscriptions(req, res, next) {
-    const subscriptions = await FollowService.getSubscriptions(req.user.id, next);
-    req.serializer = new CollectionSerializer(subscriptions, UserSerializer, req);
+    const subscriptions = await FollowService.getSubscriptions(req.user.id, req.params.count, next);
+    req.serializer = new CollectionSerializer(subscriptions, { serializerType: UserSerializer });
```
Comment (Konstantsin) [inline src/controllers/FollowController.js:52]: Difficult to understand what `count` means until you read the code to the place where it is used. For example, the name `perPage` will direct us to pagination right away.

File: Dockerfile [hunk 3]
```diff
@@ -2,10 +2,12 @@ FROM node
 
 WORKDIR /app
 
-COPY . .
+COPY package.json /app
```
[the remark this answers is not in the document]
Comment (Aretemal) [reply Dockerfile:5]: done

File: src/controllers/FollowController.js (hunk 2 above)
Comment (Aretemal) [reply src/controllers/FollowController.js:52]: done

**Pull request merged**